extension/package.json

```diff
@@ -23,7 +23,9 @@
     "orchestrator",
     "apphost"
   ],
-  "activationEvents": [],
+  "activationEvents": [
+    "workspaceContains:**/*AppHost*"
```

Comment:
The aspire extension should automatically activate when an AppHost folder is inside the workspace. This will also be necessary for future improvements.